### ISSUE TITLE
lisk_snapshot.sh does not check for failure of the snapshotting process

### ISSUE THREAD

**Issue by fchavant** [LiskHQ/lisk-scripts#109]
Blocked by: https://github.com/LiskHQ/lisk/pull/2260

---

Snapshotting process failed with:

```
[inf] 2018-07-12 08:42:38 | Rebuilding accounts states, current round: 1954, height: 197254
[ERR] 2018-07-12 08:42:38 | Snapshot creation failed - {"name":"error","length":105,"severity":"PANIC","code":"XX000","file":"pg_sema.c","line":"421","routine":"PGSemaphoreUnlock"}
[inf] 2018-07-12 08:42:38 | Cleaning up...
[inf] 2018-07-12 08:42:38 | Cleaned up successfully
```

And yet `lisk_snapshot.sh` finishes successfuly:

```
2018-07-12 08:43:05 Snapshot verification process completed
2018-07-12 08:43:05 Deleting data on table 'peers' of database 'lisk_snapshot'
2018-07-12 08:43:05 Dumping snapshot database to gzip file
2018-07-12 08:44:20 Cleaning up
2018-07-12 08:44:21 Snapshot Complete
```

Bug likely introduced in https://github.com/LiskHQ/lisk-scripts/pull/82#issuecomment-393473308

**Comment by fchavant** [comment]
Superseded by https://github.com/LiskHQ/lisk/issues/3109
